doc/rados/operations/crush-map-edits.rst

```diff
@@ -159,7 +171,7 @@ bucket types. Enter ``type`` followed by a unique numeric ID and a bucket name.
 By convention, there is one leaf bucket and it is ``type 0``;  however, you may
 give it any name you like (e.g., osd, disk, drive, storage, etc.)::
```

Comment:
@zdover23 I would remove the _etc._

Reply:
Agreed, with `e.g.` in there it's redundant.

Reply:
I have fixed this in https://github.com/ceph/ceph/pull/48362.